app/services/ams/asset_destroyer.rb

```diff
@@ -64,6 +64,8 @@ def destroy_in_postgres(asset_id)
           .with_step_args('work_resource.delete' => { user: user },
                           'work_resource.delete_all_file_sets' => { user: user })
           .call(asset_resource).value!
+      rescue Valkyrie::Persistence::ObjectNotFoundError
+        puts "No AssetResource found with ID #{asset_id}"
```

Comment:
Would a Rails logger be better here than a puts?

Reply:
In principle, yes. I chose to use `puts` in this PR because it follows the established pattern elsewhere in the file. `puts` is also visible when running this in the rails console, whereas I don't think `Rails.logger` is. 

However, a dedicated logger would be the ideal, and I may come back and rework this file to use one. But for the scope of this PR, I think leaving it as `puts` for now is ideal.

Reply:
Since this PR has been here for awhile, I went ahead and implemented the dedicated logger: b620521